docs2/site/docs/migrations/migration4.md

```diff
@@ -254,6 +254,12 @@ protected override IExecutionStrategy SelectExecutionStrategy(ExecutionContext c
 }
 ```
 
+### DataLoader Moved to Separate Project
+
+The implementation for data loaders, contained within the `GraphQL.DataLoader` namespace, has been moved into the
+[`GraphQL.DataLoader`](https://www.nuget.org/packages/GraphQL.DataLoader) NuGet package. Please import the NuGet
+package if you use data loaders. No code changes are necessary.
```

Comment:
If you're referring to this, I'm using it as an adjective/common noun separately from a proper noun.  When used as a proper noun, it is capitalized as DataLoader, and when used as an adjective (data) and common noun (loader) it is written as "data loader".

I did change the capitalization on the package tags as requested.

I see you approved the PR so I'll merge it.  It is easy enough to change the documentation if you think we should.

Reply:
Ok